src/components/shared/LanguageSwitcher/LanguageSwitcher.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import { UnstyledButton, Menu, Image, Group } from '@mantine/core';
+import en from 'components/shared/LanguageSwitcher/Image/english.png';
+import ru from 'components/shared/LanguageSwitcher/Image/russian.png';
+import classes from 'components/shared/LanguageSwitcher/LanguageSwitcher.module.css';
+import { useTranslation } from 'react-i18next';
+import { langSwitcher } from 'Enum/Enum';
+
+const data = [
+  { label: 'Русский', image: ru, code: 'ru' },
+  { label: 'English', image: en, code: 'en' },
+];
+
+function LanguageSwitcher() {
```

Comment:
use arrow function and throughout app
```suggestion
export const LanguageSwitcher: FC = () => {
```